### PR TITLE
doc: move estimator guide to workflows

### PR DESCRIPTION
**Pull request by jakmeier**
Since we now have a workflow chapter, the section on how to run the
estimator belong there.

Also added a pointer to #8031 as an example for gas parameter analysis.

### CODE REVIEW

File: docs/architecture/gas/estimator.md [hunk 1]
```diff
@@ -20,100 +20,9 @@ clients.
 The estimator code is part of the nearcore repository in the directory
 [runtime/runtime-params-estimator](https://github.com/near/nearcore/tree/master/runtime/runtime-params-estimator).
```
Comment (akhi3030) [inline docs/architecture/gas/estimator.md:21]: not part of the PR but should be update the link here to not point to master like we do for other links?

File: docs/architecture/gas/estimator.md [hunk 2]
```diff
@@ -20,100 +20,9 @@ clients.
...
-can cause severe system instability.
-
-Our current strategy is to run estimations
-with two different metrics and do so on standardized cloud hardware. The output
-is then sanity checked manually by several people. Based on that, the final gas
-parameter value is determined. Usually it will be the higher output of the two
-metrics rounded up. More details on the process will be added to this document
-in due time.
+For a practical guide on how to run the estimator, please take a look at
+[Running the Estimator](/practices/workflows/gas_estimations.html) in the
```
Comment (akhi3030) [inline docs/architecture/gas/estimator.md:24]: In https://github.com/near/nearcore/blob/ca284e1e73078c7e61f0ee5d12e00b500642fb3a/docs/architecture/gas/estimator.md, this link is dead.  I wonder if we need to do the following or it's dead because the PR is not merged yet.
```suggestion
[Running the Estimator](../../practices/workflows/gas_estimations.html) in the
```

**Review by akhi3030 — APPROVED** [review]

File: docs/architecture/gas/estimator.md (hunk 1 above)
Comment (akhi3030) [reply docs/architecture/gas/estimator.md:21]: Sounds good.

File: docs/architecture/gas/estimator.md (hunk 2 above)
Comment (jakmeier) [reply docs/architecture/gas/estimator.md]: seems to work now, with the `.md ` extension and relative paths it seems to work locally and also in the PR
